ui/src/main/java/edu/wpi/grip/ui/DeployController.java

```diff
@@ -150,9 +157,12 @@ private void deploy(String address, String user, String password, String javaHom
             StringWriter projectWriter = new StringWriter();
             project.save(projectWriter);
 
-            // Upload the GRIP core JAR and the serialized project to the robot
-            scp.upload(new StringInMemoryFile(PROJECT_FILE, projectWriter.toString()), deployDir + "/");
-            scp.upload(new FileSystemFile(LOCAL_GRIP_JAR), deployDir + "/" + GRIP_JAR);
+            // Upload the GRIP core JAR, a wrapper script to run it, and the serialized project to the robot
+            final String commandStr = command.get();
```

Comment:
Matched quotes are fine, since it's executed as a bash command.  We need to allow quotes so you can provide an argument with spaces in it.